src/main/resources/templates/misc/compare.html

```diff
@@ -27,7 +53,22 @@
               </div>
               <div th:replace="~{fragments/common :: fileSelector(name='fileInput', multiple=false, accept='application/pdf', remoteCall='false')}"></div>
               <div th:replace="~{fragments/common :: fileSelector(name='fileInput2', multiple=false, accept='application/pdf', remoteCall='false')}"></div>
+
+              <div class="row">
+                <div class="flex-container">
+                  <div class="color-selector spacer1">
+                    <label>Highlight Color 1: </label>
```

Comment:
must support custom languages like having it as 

th:text="#{compare.highlightColor}"

and
compare.highlightColor=Highlight Color 1:

in languages GB file

Reply:
Fixed

Reply:
My latest commit changes the messages file for the US, do I need to do GB as well?

Reply:
Commit 5 added for US message file, just updated the GB one in commit 6.